apis/types/error.go

```diff
@@ -20,24 +20,24 @@ type Error struct {
 }
 
 // Validate validates this error
-func (m *Error) Validate(formats strfmt.Registry) error {
-	return nil
+func (m *Error) Validate(formats strfmt.Registry) (err error) {
```

Comment:
Why did you revise there? Using the anonymous return value in the signature of the function is ok, especially we didn't add significant logics.